themes/base.theme.bash

```diff
@@ -98,6 +98,44 @@ function scm_prompt_info {
   [[ $SCM == $SCM_SVN ]] && svn_prompt_info && return
 }
 
+function scm_prompt_status {
```

Comment:
Can you please add a comment above this function to explain how it differs from the existing one that provides the full output?